install/do_realms.php

```diff
@@ -47,9 +47,9 @@ public function __construct()
                 echo json_encode(array("success" => false, "message" => "Looks like your world database doesn't exist"));
                 exit();
             }
-			
+
             $query = ("INSERT INTO realms(`id`, `hostname`, `username`, `password`, `char_database`, `world_database`, `cap`, `realmName`, `console_username`, `console_password`, `console_port`, `emulator`, `realm_port`, `override_port_world`, `override_username_world`, `override_password_world`, `override_hostname_world`, `override_port_char`, `override_username_char`, `override_password_char`, `override_hostname_char`)
-                        VALUES('?',
+                        VALUES('1',
```

Comment:
This doesn't seem like a good idea

Reply:
I know..

but seems the default mysql server config can't handle that

but shouldn't be a problem

ID 1 should never exist because the DB is freshly imported

Reply:
Yeah but the auto-increment value will be wrong so when you try to create a new realm it'll try to create 1 and it'll already exist. At the very least you'd need to manually set that to 2, but I'm confident *SQL can deal with this

Reply:
If you drop this one from here or adopt the same change from the other PR I'd be good with this PR